WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsViewModel.swift

```diff
@@ -416,6 +423,15 @@ private extension WooShippingCreateLabelsViewModel {
         }
     }
 
+    /// Loads shipment info from remote and creates view model for split shipments.
+    ///
+    @MainActor
+    func loadShipmentsInfo() async {
+        splitShipmentsViewModel = WooShippingSplitShipmentsViewModel(siteID: order.siteID,
+                                                                     orderID: order.orderID,
+                                                                     stores: stores)
+    }
```

Comment:
Is this also WIP? Should we disable splitting if an order has only one item?

Reply:
Good point. Yes, this is WIP. 

We need to load the shipment info from the remote and show this "Split shipments" view. I will update this after adding endpoint support in https://github.com/woocommerce/woocommerce-ios/issues/15370.